hack/test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+junitFilenamePrefix() {
+  if [[ -z "${JUNIT_REPORT_DIR}" ]]; then
+    echo ""
+    return
+  fi
+  mkdir -p "${JUNIT_REPORT_DIR}"
+  DATE=$( date +%s | base64 | head -c 15 )
+  echo "${JUNIT_REPORT_DIR}/junit_$DATE"
+}
+
+function produce_junit_xmlreport {
+  local -r junit_filename_prefix=$1
+  if [[ -z "${junit_filename_prefix}" ]]; then
+    return
+  fi
+
+  local junit_xml_filename
+  junit_xml_filename="${junit_filename_prefix}.xml"
+
+  if ! command -v gotestsum >/dev/null 2>&1; then
+    echo "gotestsum not found; installing now"
+    go install gotest.tools/gotestsum@latest
```

Comment:
instead of `@latest` can we stick to some released tag to be produce consistent results.

Reply:
I shall point to the last release `v1.8.2` and re-run.